corehq/apps/reports/standard/monitoring.py

```diff
@@ -310,43 +343,99 @@ def make_column(title, help_text, num_days):
     @property
     @memoized
     def selected_users(self):
-        return _get_selected_users(self.domain, self.request)
+        users = _get_selected_users(self.domain, self.request)
+        return [
+            self.RowData(
+                id=user.user_id,
+                name=user.raw_username,
+                name_in_report=user.username_in_report,
+                filter_func=CaseListFilter.for_user
+            ) for user in users
+        ]
+
+    @property
+    def has_case_group_filters(self):
+        slugs = EMWF.get_value(self.request, self.domain)
+        filter_count = len(EMWF.selected_group_ids(slugs) + EMWF.selected_location_ids(slugs))
+        return filter_count > 0
+
+    @property
+    @memoized
+    def selected_groups(self):
+        slugs = EMWF.get_value(self.request, self.domain)
```

Comment:
nit: we need slugs only if `self.has_case_group_filters` is true? So, it can move into the if block?

Reply:
[c0293ae](https://github.com/dimagi/commcare-hq/pull/34464/commits/c0293ae0e09175607011de1a3114ad03186553d5)